qutebrowser/browser/commands.py

```diff
@@ -822,22 +876,36 @@ def zoom(self, zoom: int = None, count=None):
         message.info("Zoom level: {}%".format(level), replace=True)
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def tab_only(self, prev=False, next_=False):
+    def tab_only(self, prev=False, next_=False, force=False):
         """Close all tabs except for the current one.
 
         Args:
             prev: Keep tabs before the current.
             next_: Keep tabs after the current.
+            force: Avoid confirmation for pinned tabs.
         """
         cmdutils.check_exclusive((prev, next_), 'pn')
         cur_idx = self._tabbed_browser.currentIndex()
         assert cur_idx != -1
 
+        def _to_close(i):
```

Comment:
The reason for this is to avoid having all the logic that is currently in `_to_close` in both the upper and lower for loops below. I can duplicate the variable set, but I think that would be more messy, thoughts?